contents/handbook/cs-and-onboarding/renewals.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Renewals
+sidebar: Handbook
+showTitle: true
+---
+
+### Renewal principals
+Being on an annual plan is a win-win solution for both PostHog and the customer. Customers get discounts on the credits they purchase and PostHog gets confirmed revenue.
+
+When estimating renewal amount, we want to make sure we accurately determine the amount of credits the customer will need in the next 12 months. This is not an opportunity to upsell.
+
+This page walks through recommendations for approaching and handling renewals. [Contract rules](/handbook/growth/sales/contract-rules) and [how to create contracts](/handbook/growth/sales/contracts) are covered in relevant pages under our shared processes. 
+
+## When to start
+Start renewal conversations 2 months before the contract renewal date. Vitally and Slack will keep you on track with automated reminders. When a customer hits the 2-month mark, they'll automatically enter the `Upcoming renewal` segment, you'll get a task assigned to you in Vitally, and Slack will send a notification.
```

Comment:
2 months feels like leaving it very late, but is the assumption here that you are already in frequent contact with the customer? (I'd be very nervous if they've been quiet a lot and we only start talking renewal 2 months out, so maybe it's a bit context dependent?)